UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentAncillaryFilesTest.cs

```diff
@@ -0,0 +1,105 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using UKHO.ExchangeSetService.Common.Configuration;
+using UKHO.ExchangeSetService.Common.Helpers;
+using UKHO.ExchangeSetService.Common.Models.SalesCatalogue;
+using UKHO.ExchangeSetService.FulfilmentService.Services;
+
+namespace UKHO.ExchangeSetService.Webjob.UnitTests.Services
+{
+    [TestFixture]
+    public class FulfilmentAncillaryFilesTest
+    {
+        public ILogger<FulfilmentDataService> fakeLogger;
+        public IOptions<FileShareServiceConfiguration> fakefileShareServiceConfig;
+        public FulfilmentAncillaryFiles fulFilmentAncillaryFilesTest;
+        public IFileSystemHelper fakeFileSystemHelper;
+        public string fakeExchangeSetInfoPath = @"C:\\HOME";
+        public string batchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
```

Comment:
Kindly change batchId to fakeBatchId

Reply:
As per the Comment i have done changes batchId to fakeBatchId